<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainInterface</class>
 <widget class="QMainWindow" name="MainInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainInterface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>110</width>
      <height>330</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>17</number>
     </property>
     <item>
      <widget class="QPushButton" name="button1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #f6f7fa, stop:1 #dadbde);
  border: 1px solid #bdc3c7;
  border-radius: 5px;
  padding: 5px;
  font-size: 14px;
  text-align: center;
}

QPushButton:hover {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #e0e0e0, stop:1 #cecece);
}

QPushButton:pressed {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #dadbde, stop:1 #f6f7fa);
  padding-left: 5px;
  padding-top: 5px;
}

QPushButton:disabled {
  background-color: #f6f7fa;
  border: 1px solid #bdc3c7;
  color: #bdc3c7;
}
</string>
       </property>
       <property name="text">
        <string>过往单词</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #f6f7fa, stop:1 #dadbde);
  border: 1px solid #bdc3c7;
  border-radius: 5px;
  padding: 5px;
  font-size: 14px;
  text-align: center;
}

QPushButton:hover {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #e0e0e0, stop:1 #cecece);
}

QPushButton:pressed {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #dadbde, stop:1 #f6f7fa);
  padding-left: 5px;
  padding-top: 5px;
}

QPushButton:disabled {
  background-color: #f6f7fa;
  border: 1px solid #bdc3c7;
  color: #bdc3c7;
}
</string>
       </property>
       <property name="text">
        <string>自定义壁纸</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #f6f7fa, stop:1 #dadbde);
  border: 1px solid #bdc3c7;
  border-radius: 5px;
  padding: 5px;
  font-size: 14px;
  text-align: center;
}

QPushButton:hover {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #e0e0e0, stop:1 #cecece);
}

QPushButton:pressed {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #dadbde, stop:1 #f6f7fa);
  padding-left: 5px;
  padding-top: 5px;
}

QPushButton:disabled {
  background-color: #f6f7fa;
  border: 1px solid #bdc3c7;
  color: #bdc3c7;
}
</string>
       </property>
       <property name="text">
        <string>自定义资源</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #f6f7fa, stop:1 #dadbde);
  border: 1px solid #bdc3c7;
  border-radius: 5px;
  padding: 5px;
  font-size: 14px;
  text-align: center;
}

QPushButton:hover {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #e0e0e0, stop:1 #cecece);
}

QPushButton:pressed {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #dadbde, stop:1 #f6f7fa);
  padding-left: 5px;
  padding-top: 5px;
}

QPushButton:disabled {
  background-color: #f6f7fa;
  border: 1px solid #bdc3c7;
  color: #bdc3c7;
}
</string>
       </property>
       <property name="text">
        <string>选择语言</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #f6f7fa, stop:1 #dadbde);
  border: 1px solid #bdc3c7;
  border-radius: 5px;
  padding: 5px;
  font-size: 14px;
  text-align: center;
}

QPushButton:hover {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #e0e0e0, stop:1 #cecece);
}

QPushButton:pressed {
  background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                    stop:0 #dadbde, stop:1 #f6f7fa);
  padding-left: 5px;
  padding-top: 5px;
}

QPushButton:disabled {
  background-color: #f6f7fa;
  border: 1px solid #bdc3c7;
  color: #bdc3c7;
}
</string>
       </property>
       <property name="text">
        <string>导出词典</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>470</width>
      <height>330</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4"/>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="ignore">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    /* 默认状态下的样式 */
	border:0px;
    image: url(:/res/ignore1.png);
}

QPushButton:hover {
    /* 鼠标悬停时的样式 */
	border:0px;
    image: url(:/res/ignore2.png);
}

QPushButton:pressed {
    /* 按钮被按下时的样式 */
	border:0px;
	image: url(:/res/ignore2.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    /* 默认状态下的样式 */
	border:0px;
    image: url(:/res/error1.png);
}

QPushButton:hover {
    /* 鼠标悬停时的样式 */
	border:0px;
    image: url(:/res/error2.png);
}

QPushButton:pressed {
    /* 按钮被按下时的样式 */
	border:0px;
	image: url(:/res/error2.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 300 22pt &quot;Bahnschrift Light&quot;;</string>
    </property>
    <property name="text">
     <string>QuickTranslateAdmin</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
